docs/adr/0001-mobile-app-architecture.md

```diff
@@ -0,0 +1,107 @@
+# Build Relay Mobile App with [Mobile Web | Qt | React Native | WebView ]
+
+- Status: proposed
+- Deciders: Luke Crouch, Emmett Lynch, Maxx Crawford
+- Date: pending
+
+Technical Spike:
+
+- [MPP-2633](https://mozilla-hub.atlassian.net/browse/MPP-2633)
+
+## Context and Problem Statement
+
+Make it just as easy to use your Firefox Relay phone number as your real phone number.
+See Google Voice as an example.
+[Firefox Relay Mobile Problems doc](https://docs.google.com/document/d/14wYnem-oPPA4_tjp018U3SyHpcdyKodDBbreRBkDd24/edit)
+
+## Decision Drivers
+
+- We are concerned that the Relay Phone experience is causing churn.
```

Comment:
These descriptions seem to be an attempt to describe why we'd build a mobile app in the first place, rather than that we're looking to pick the implementation technology - which the rest of the ADR appears to be about.

Here's something focused more on the technology choice, just to kick off a discussion:

```suggestion
## Context and Problem Statement

We might need to support features on mobile that cannot be provided via the browser. We don't have anyone with (native) mobile-specific experience on the Relay team; the VPN team does have a QML app that supports Android and iOS, among others.

## Decision Drivers

- Cost of quickly validating our assumptions about the added value of a mobile app.
- How far we can get with the expertise we have.
```

(It's not quite clear to me what the goal of the "decision drivers" section is, and googling didn't help much, so I'm not sure if these are helpful.)